pom.xml

```diff
@@ -105,10 +114,9 @@
       <fuse.mqtt.client.version>1.16</fuse.mqtt.client.version>
       <caffeine.version>3.1.8</caffeine.version>
       <guava.version>33.2.1-jre</guava.version>
-      <hawtio.version>2.17.7</hawtio.version>
+      <hawtio.version>4.0.0-RC1</hawtio.version>
```

Comment:
The console build is using 4.0.0 it appears, so seems like this should be updated to at least that? (There is also a 4.1.0 now)

Reply:
Or actually this seems to not be needed at all. It only appears used in a bit of commented out dep in artemis-console that I previously commented on, and in leftover old module poms that are no longer being built and should be deleted: `artemis-hawtio/pom.xml`, `artemis-hawtio/activemq-branding/pom.xml`, `artemis-hawtio/artemis-plugin/pom.xml`